cloudwatch_dashboard_rds/main.tf

```diff
@@ -262,8 +262,8 @@ resource "aws_cloudwatch_dashboard" "main" {
                 "view": "timeSeries",
                 "stacked": false,
                 "metrics": [
-                    [ "AWS/RDS", "NetworkTransmitThroughput", "DBInstanceIdentifier", "${var.db_instance_identifier}" ],
-                    [ ".", "NetworkReceiveThroughput", ".", "." ]
+                    [ "AWS/RDS", "NetworkTransmitThroughput", "DBInstanceIdentifier", "${var.db_instance_identifier}", { "stat": "Maximum" } ],
```

Comment:
**question:** was this the one that @pauldoomgov wanted to change the units to bits/sec from bytes/sec? Is that something we can do here?

Reply:
It's NBD.   I should drop in a PR to just add a line showing the limit.